src/SentryFullStory.ts

```diff
@@ -41,6 +41,17 @@ class SentryFullStory {
           }
         };
 
+        let name = '';
+        let message = '';
+        const isError = (exception: string | Error): exception is Error => {
```

Comment:
@patrick-fs I think this belongs in the util file to keep things cleaner here

Reply:
Or better yet, just make a utility function that returns the extra properties from the error that we can put into `FullStory.event` call with the spread operator.

Reply:
done.